cmd/epp/runner/runner.go

```diff
@@ -219,7 +175,7 @@ func (r *Runner) Run(ctx context.Context) error {
 	setupLog.Info(r.eppExecutableName+" build", "commit-sha", version.CommitSHA, "build-ref", version.BuildRef)
 
 	// Validate flags
-	if err := validateFlags(); err != nil {
+	if err := opts.Validate(); err != nil {
```

Comment:
we should call `Validate` right after flags parsing. 
otherwise, flags might not been validated and we use it in L168 (opts.Tracing)

Reply:
True. This is the same behavior as before. I can move it up before tracing.
The flow should be: 
```go
   pflag.Parse()
   if err := opts.Complete(); err != nil { 
      ... 
   }
   if err := opts.Validate(); err != nil { 
      ... 
   }
```
The call to `opts.Complete()` is currently missing and should be added.

Reply:
yes, please 🙏